doc/PagoPa-Proxy.postman_collection.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
I'd remove this file and `PagoPaApi_Services` as they add very little value to the docs (when we'll have the swagger specs that would suffice).